tools/python/util/usage_error.py

```diff
@@ -0,0 +1,10 @@
+#  // Copyright (c) Microsoft Corporation. All rights reserved.
+#  // Licensed under the MIT License.
+from .base_error import BaseError
+
+
+class UsageError(BaseError):
```

Comment:
Was there a motivation for keeping the errors in different modules? Looks like they can all be defined in the same module.